bigchaindb_driver/driver.py

```diff
@@ -183,7 +181,11 @@ def transfer(self, transaction, *conditions, signing_key=None):
         signing_key = signing_key if signing_key else self.signing_key
         if not signing_key:
             raise InvalidSigningKey
-        transfer_transaction = transaction.transfer(list(conditions))
+        inputs = transaction.to_inputs()
```

Comment:
`get_inputs` would be arbitrary since `transaction` could be of `operation == TRANSFER`, so `get_inputs` would mean you retrieve the already spent inputs :)